examples/bin/encrypt_command_line.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# Basic example invocation of the AWS Encryption CLI to encrypt input from the command line
+# See full docs for more details:
+#  https://docs.aws.amazon.com/encryption-sdk/latest/developer-guide/crypto-cli-examples.html
+
+# Usage: encrypt_command_line.sh $PLAINTEXT $CMK_ARN
+
+echo "$1" | aws-encryption-cli --encrypt \
+                               --suppress-metadata \
+                               --input - \
+                               --output - \
+                               --encode \
+                               --wrapping-keys key=$2
```

Comment:
is there a reason that some files have a new line and some do not?

Reply:
Nope, inconsistency on my part. Fixed.